src/Stack2nix/Render.hs

```diff
@@ -88,12 +96,12 @@ renderOne args locals drv' =
            drv = drv'
                  & doCheck .~ (argTest args && isLocal)
                  & runHaddock .~ (argHaddock args && isLocal)
-                 & benchmarkDepends . haskell .~ S.empty
+                 & benchmarkDepends . haskell .~ Set.empty
                  -- find a DRY way
-                 & testDepends . haskell .~ (if (argTest args && isLocal) then (view (testDepends . haskell) drv') else S.empty)
-                 & testDepends . pkgconfig .~ (if (argTest args && isLocal) then (view (testDepends . pkgconfig) drv') else S.empty)
-                 & testDepends . system .~ (if (argTest args && isLocal) then (view (testDepends . system) drv') else S.empty)
-                 & testDepends . tool .~ (if (argTest args && isLocal) then (view (testDepends . tool) drv') else S.empty)
+                 & testDepends . haskell .~ (if (argTest args && isLocal) then (view (testDepends . haskell) drv') else Set.empty)
+                 & testDepends . pkgconfig .~ (if (argTest args && isLocal) then (view (testDepends . pkgconfig) drv') else Set.empty)
+                 & testDepends . system .~ (if (argTest args && isLocal) then (view (testDepends . system) drv') else Set.empty)
+                 & testDepends . tool .~ (if (argTest args && isLocal) then (view (testDepends . tool) drv') else Set.empty)
```

Comment:
Would be easier to write

```haskell
let 
  condition = argTest args && isLocal
  overwrite s = if condition then s else Set.empty 
in 
  drv'
    & testDepends . haskell %~ overwrite
```

But, I see that this is just a `S // Set` change, so no need to refactor :)